wiki/Help_Centre/en.md

```diff
@@ -1,3 +1,7 @@
+---
+needs_cleanup: true
```

Comment:
not sure if needed here though

Reply:
i thought this one was strange because it has so many lines for no reason
some of the language is weird later on it just needs small changes